templates/desktop/developers.html

```diff
@@ -99,7 +97,7 @@
     <div class="row">
       <div class="col-6">
         <h2>All the tools developers need</h2>
-        <p>Whether your team uses Python, Ruby, Node.js or Java, no operating system is easier to set up than Ubuntu. Everything your developers need is just a click or an <code>apt</code> away.</p>
+        <p>Whether your team uses Python, Ruby, Node.js or Java, no operating system is easier to set up than Ubuntu. Everything your developers need is just a snap or an <code>apt</code> away.</p>
```

Comment:
For consistency "snap" should be in <code> too (Update this is the copydoc as well)